lib/entities/parse_address_from_domain.dart

```diff
@@ -26,15 +27,15 @@ Future<ParsedAddress> parseAddressFromDomain(
 
     if (domainParts.length <= 1 || domainParts.first.isEmpty || name.isEmpty) {
       try {
-        final addresses = await fetchYatAddress(domain, ticker);
+        final addresses = await YatRecord.fetchYatAddress(domain, ticker);
 
         if (addresses?.isEmpty ?? true) {
           return ParsedAddress(
               addresses: [domain], parseFrom: ParseFrom.yatRecord);
         }
 
         return ParsedAddress(
-            addresses: addresses, name: domain, parseFrom: ParseFrom.yatRecord);
+            addresses: addresses.map((e) => e.address).toList(), name: domain, parseFrom: ParseFrom.yatRecord);
       } catch (e) {
         return ParsedAddress(addresses: [domain]);
       }
```

Comment:
Let change that logic a bit. For example let check is user entered text emoji or not, then if it's emoji we will use Yat for resolving the address. If the text is not emoji let check is it domain addresses or not and will use openallias for resolve address.

Reply:
done